src/libraries/System.Text.Json/src/System/Text/Json/Document/JsonElement.cs

```diff
@@ -1163,6 +1163,13 @@ internal string GetPropertyName()
             return _parent.GetNameOfPropertyValue(_idx);
         }
 
+        internal ReadOnlySpan<byte> GetRawPropertyNameAsUtf8Span()
+        {
```

Comment:
This is related to https://github.com/dotnet/runtime/issues/77666. Could you add a `// TODO` comment linking to that issue?